chart/templates/gateway-virtual-service.yaml

```diff
@@ -13,6 +13,14 @@ spec:
   gateways:
   - gsp-gsp-cluster.gsp-system
   http:
+    - match:
+      - uri:
+          exact: {{ include "gateway.host" . }}/ServiceMetadata
```

Comment:
Does the metadata url need to match the entityId of that metadata?
i.e. the entityId of https://test-metadata.london.verify.govsvc.uk/metadata.xml is "https://test-metadata.london.verify.govsvc.uk/metadata.xml". See also https://eidas.minez.nl/EidasNodeC/ConnectorMetadata
I'm not certain its required, but does seem to be a convention.

Reply:
Yes it does but we can change that later - this will publish the existing metadata at the new URL in addition to the old one. If this works, we'll change the entity IDs and remove old metadata

Reply:
Ok, makes sense